ci-operator/config/openshift/debug-network/openshift-debug-network-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
```

Comment:
Ideally not good to call `cli` `base` ... will just confuse everyone since `base` is an image

Reply:
done